### PR TITLE
Participation local menu

### PR DESCRIPTION
**Pull request by apradillap**
Closes https://github.com/PopulateTools/issues/issues/207

### What does this PR do?

Add a new template navigation_process

### How should this be manually tested?

You can personalize the template from /admin/gobierto_core/templates

and you'll see in

captura de pantalla 2018-01-12 a las 16 50 17

### CODE REVIEW

File: app/models/gobierto_core/site_template.rb [hunk 1]
```diff
@@ -6,5 +6,22 @@ module GobiertoCore
   class SiteTemplate < ApplicationRecord
     belongs_to :template
     belongs_to :site
+
+    def self.current_site_has_custom_template?(site, template_path)
+      GobiertoCore::SiteTemplate.current_site_custom_template(site, template_path).any?
+    end
+
+    def self.current_site_custom_template(site, template_path)
+      GobiertoCore::SiteTemplate.joins(:template).where("gcore_site_templates.site_id = ? AND gcore_templates.template_path = ?",
```
Comment (ferblape) [inline app/models/gobierto_core/site_template.rb]: Could we use `self.class.table_name`?

File: app/views/gobierto_participation/layouts/_navigation.sub.html.erb [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-  <div class="sub-nav-item">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.questions'), '#' %>
-  </div>
-  <div class="sub-nav-item">
-    <%= link_to t('gobierto_participation.layouts.menu_subsections.ideas'), '#' %>
-  </div>
+  <% liquid_path = "gobierto_participation/layouts/navigation_process.liquid" %>
+  <% liquid_str = GobiertoCore::SiteTemplate.liquid_str(current_site, liquid_path) %>
+
+  <%= raw render_liquid(liquid_str) %>
```
Comment (ferblape) [inline app/views/gobierto_participation/layouts/_navigation.sub.html.erb]: I'd refactor it a bit, too many things to remember when trying to render a template.

I'd like the syntax to be something like: `<%= raw render_liquid("gobierto_participation/layouts/navigation_process") %>

- remove the calls to liquid_str from the view, and move them to the helper
- remove the extension `.liquid` of the template

File: app/controllers/gobierto_admin/gobierto_core/templates_controller.rb [hunk 3]
```diff
@@ -5,8 +5,8 @@ module GobiertoCore
     class TemplatesController < BaseController
       def index
         template = ::GobiertoCore::Template.first
-        if current_site_has_custom_template?(template.template_path)
-          @site_template = current_site_custom_template(template.template_path).first
+        if ::GobiertoCore::SiteTemplate.current_site_has_custom_template?(current_site, template.template_path)
```
Comment (ferblape) [inline app/controllers/gobierto_admin/gobierto_core/templates_controller.rb]: This is duplicated in `index` and `edit` actions. Could we extract it to a method or filter and remove the duplication?

**Review by ferblape — CHANGES_REQUESTED** [review]
Looks good but it could be refactored a bit to simplify it.

**Review by furilo — APPROVED** [review]

**Review by ferblape — APPROVED** [review]
👍

**Pull request merged**